modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -1,123 +1,83 @@
-
-=== Limitations
+. Make sure you have the latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
 
-[NOTE]
-====
-These baseline limits are subject to change after the Serverless beta release.
-====
+== Limits
 
-Each Serverless cluster has the following limits:
+Each Serverless account has the following limits:
 
 * Ingress: up to 10 MBps, 0.5 MBps guaranteed
 * Egress: up to 30 MBps, 1.5 MBps guaranteed
 * Partitions: 100
-* Max message size: 20 MB
-* Max retention: unlimited
+* Message size: 20 MB
+* Retention: unlimited time, 10 GiB size per topic
 * Storage: unlimited
 * Users: 30
 * ACLs: 120
 * Consumer groups: 200
 
-Redpanda Serverless is opinionated about Kafka configurations. For example, automatic topic creation is disabled. Some systems expect the Kafka service to automatically create topics when a message is produced to a topic that doesn't exist. You can create topics in the Redpanda Cloud *Topics* page or with `rpk topic create`.
-
-== Free trial 
-
-Start using Redpanda Serverless beta by joining the free trial. To ask questions, give feedback, or extend your trial, use the *#serverless* https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
-
-=== Prerequisites
-
-. https://redpanda.com/try-redpanda?section=cloud-trial[Sign up for a beta trial^] of Redpanda Serverless.
-
-. Make sure you have the latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
+[NOTE]
+====
+These baseline limits are subject to change. To increase your limits, contact https://support.redpanda.com/hc/en-us/requests/new[support^]. 
+====
 
-=== Credits and limits
+== Trial credits
 
-Each beta trial supports five Serverless clusters. Redpanda issues 100 free credits, which is enough to run Redpanda with reasonable throughput. These credits expire in 14 days. Usage is based on the following:
+New trials receive $100 (USD) in free credits to spend in the first 14 days. This should be enough to run Redpanda with reasonable throughput. No credit card is required for a trial. To continue using Serverless after your trial expires, you can enter a credit card and pay as you go. Any remaining credit balance is used before you are charged. 
 
-* Ingress: 0.1 credit per GB in (writes)
-* Egress: 0.1 credit per GB out (reads)
-* Partitions: 0.003 credit (metered in 10 min fractions) per partition/hour	
-* Storage: 0.1 credit (metered in 10 min fractions) per GB stored/month
+When either the 100 credits expire or the 14 days in the trial expire, the clusters move into a suspended state, and you won't be able to access your data in either the Redpanda Cloud UI or with the Kafka API. There is a 7 day grace period following the end of the trial. After that, the data is permanently deleted. 
```

Comment:
I thought we deleted things after two weeks? @vladoschreiner can you confirm this? Because I believe the UI says two weeks.

Reply:
Email also shows 14 days. @vladoschreiner can you please confirm? 
![image](https://github.com/redpanda-data/docs/assets/107008679/3178387b-55b9-420a-83e6-717d39b24285)